web/tests/help-page.spec.ts

```diff
@@ -0,0 +1,57 @@
+//Tests the help page
+import { test, expect, Page } from '@playwright/test';
+
+async function checkExactlyOneCurrent(page: Page) {
+    // There is only one highlighted item:
+    await expect(page.locator('span.find-highlight-current')).toHaveCount(1);
+}
+
+async function checkCurrentHighlight(page: Page, expectedPos) {
+    await checkExactlyOneCurrent(page);
+    // Check correct item:
+    const cssQuery = 'span.find-highlight.find-highlight-current.find-highlight-' + expectedPos;
+    await expect(page.locator(cssQuery)).toBeVisible();
+}
+
+test('search box in help', async ({ page }) => {
+    await page.goto('#/help');
+    const field = page.locator('#find-in-page-text');
+    const help = page.locator('#whole-help');
+    expect(help.evaluate(e => e.scrollTop)).toBe(0);
```

Comment:
This needs an `await`. `evaluate` returns a promise. I think you can do `await expect(promise).toBe(whatever)`. But if that doesn't work, then you have to do `expect(await promise).toBe(whatever)`.

If you have the patience to look into our ESLint setup, this is the rule that would help catch these mistakes: [`no-floating-promises`](https://typescript-eslint.io/rules/no-floating-promises/)

Reply:
> Looks like I'm deleting it in #341. 
I think you said to delete those on merge.